homeassistant/components/tado/entity.py

```diff
@@ -11,8 +11,8 @@ def __init__(self, device_info):
         """Initialize a Tado device."""
         super().__init__()
         self._device_info = device_info
-        self.device_name = device_info["shortSerialNo"]
-        self.device_id = device_info["serialNo"]
+        self.device_name = device_info["serialNo"]
+        self.device_id = device_info["shortSerialNo"]
```

Comment:
Is this changing the device id?

Reply:
At least on my devices, `serialNo` and `shortSerialNo` are identical.
However, the following documentation of the Tado API specifies that {{device_short_id}} should be used when accessing a device based on its ID:
https://documenter.getpostman.com/view/154267/S11Bz2gw#ec44797a-9aff-4acb-9095-b751f7216e1e

Reply:
Take note that there is not official API documentation, so we can't really know what value should be used where.

Reply:
> Take note that there is not official API documentation, so we can't really know what value should be used where.

@michaelarnauts you’re right, but since there’s no official API documentation...

Reply:
Code owners should approve this change. The rest looks good.